src/articleView-components/CommentsContainer.jsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getCommentsByArticleId } from "../api";
+import { Comment } from "./Comment";
+
+export function CommentsContainer({ article }) {
+  const [comments, setComments] = useState([]);
+  const { article_id } = useParams();
+  useEffect(() => {
+    getCommentsByArticleId(article_id)
+      .then((comments) => {
+        setComments(comments);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }, []);
+  return (
+    <div className="comments">
```

Comment:
What if there are no comments? What do you want the user to see? Article 37 has none if you want to test.